packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -822,10 +816,7 @@ export class TokenDetectionController extends StaticIntervalPollingController<To
         .getMultiNetworksBalances(selectedAddress, chainIds, supportedNetworks)
         .catch(() => null);
 
-      if (
-        !tokenBalancesByChain ||
-        Object.keys(tokenBalancesByChain).length === 0
-      ) {
+      if (tokenBalancesByChain === null) {
```

Comment:
This used to return `failed` if the api response was empty.  But that's a valid response if it's an empty account with no tokens.